app-frontend/packs/components/dashboard/homePage/HomePageWithBanners.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+
+    <section class="tt-home-page-with-banners">
+
+        <HomePageBanner
+            v-for="banner of banners"
+            :key="banner"
+            :banner="banner"
+            @onDeleteBanner="onDeleteBanner"
+        />
+
+            <Button
+                class="tt-page__button tt-page__button--first"
+                :type="'button'"
+                @onClick="this.$emit('create')"
+            >
+                Create new banner
+            </Button>
+
+    </section>
+
+    <Confirm
+        v-if="isShowPopup"
+        @confirm="isShowPopup = false"
```

Comment:
логику обычно выносят в методы, не важно одна строка тут или нет. Так как логика может увеличиться 

```suggestion
        @confirm="closePopup"
```

Reply:
виправив